webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -373,9 +376,9 @@ export function createDefaultIntroMessage(
                         dialogType={AddGroupsToTeamModal}
                         dialogProps={{channel}}
                     >
-                        <LocalizedIcon
+                        <i
                             className='fa fa-user-plus'
-                            title={{id: t('generic_icons.add'), defaultMessage: 'Add Icon'}}
+                            title={intl.formatMessage({id: 'generic_icons.add', defaultMessage: 'Add Icon'})}
```

Comment:
Can you remove the title from here because of the button already contains the text "add other groups to this team" so the screen readers will read that and and might also announce the icon which we don't want to. Sorry for the re write in this 🥲

Reply:
Will do thanks👍